docs/CONFIGURATION.md

```diff
@@ -1,23 +1,30 @@
-
 # Configuration
 
 Ensure you have [setup your environment and configuration file](SETUP.md) before continuing,
 
-**WARNING**: The configuration file `config.json` files will contain sensitive information. Do not share this information. 
+**WARNING**: The configuration file `config.json` files will contain sensitive information. Do not share this information.
 
 ## Where to get the data
 
 The listed scopes are relevant for the existing commands in the repository. If you would like to add new features with more in-depth API interactions then you should adjust the scopes accordingly.
 
 ### Twitch
 
-To get the auth_code construct your URL and enter it into the browser.  Here's an example:
+Method 1: Manual URL Construction
+
+To get the auth_code construct your URL and enter it into the browser. Here's an example:
 
 `https://id.twitch.tv/oauth2/authorize?response_type=code&client_id=<your_client_id>&redirect_uri=<your_redirect_uri>&scope=channel%3Amanage%3Aredemptions+channel%3Aread%3Aredemptions+moderator%3Amanage%3Abanned_users+chat%3Aread+chat%3Aedit+moderator%3Aread%3Achatters+channel%3Amanage%3Abroadcast`
 
 Once you open this and authorize access, it will redirect you. Take the auth code from the new URL.
 
-The required scopes are `chat:read`, `chat:edit`, `channel:manage:redemptions`, `channel:read:redemptions`, `moderator:manage:banned_users` and `channel:manage:broadcast`. 
+The required scopes are `chat:read`, `chat:edit`, `channel:manage:redemptions`, `channel:read:redemptions`, `moderator:manage:banned_users` and `channel:manage:broadcast`.
```

Comment:
Let's move this outside of the Method 1 part as it is relevant point for both methods